java/util/src/test/java/com/google/protobuf/util/JsonFormatTest.java

```diff
@@ -659,7 +659,9 @@ public void testMapNullValueIsRejected() throws Exception {
     } catch (InvalidProtocolBufferException e) {
       // Exception expected.
     }
+  }
 
+  public void testMapEnumNullValueIsIgnored() throws Exception {
```

Comment:
Please keep the old "testMapNullValueIsRejected()" method as-is, and add a new "testMapEnumNullValueIsIgnored()" method.